chat-android/src/main/java/com/ably/chat/ChatApi.kt

```diff
@@ -15,6 +15,7 @@ import kotlinx.coroutines.suspendCancellableCoroutine
 private const val API_PROTOCOL_VERSION = 3
 private const val PROTOCOL_VERSION_PARAM_NAME = "v"
 private val apiProtocolParam = Param(PROTOCOL_VERSION_PARAM_NAME, API_PROTOCOL_VERSION.toString())
+private const val ENDPOINT_VERSION = PROTOCOL_VERSION_PARAM_NAME + API_PROTOCOL_VERSION
```

Comment:
we should be careful here, chat api version != core sdk api version, I'm not sure we should use`/chat/v3` (I haven't seen spec change or DR about this and it's out of scope of this PR).

Reply:
sorry missed that we moved to v3, but it still separate protocol version, I suggest something like this:

```kotlin
private const val CHAT_PROTOCOL_VERSION = 3
private const val PUB_SUB_PROTOCOL_VERSION = 3
private const val PROTOCOL_VERSION_PARAM_NAME = "v"
private val pubSubProtocolParam = Param(PROTOCOL_VERSION_PARAM_NAME, PUB_SUB_PROTOCOL_VERSION.toString())
private const val CHAT_PATH_PROTOCOL_VERSION = PROTOCOL_VERSION_PARAM_NAME + CHAT_PROTOCOL_VERSION
```

Reply:
Updated -> https://github.com/ably/ably-chat-kotlin/pull/124/commits/ee273d32357f9288d2c9f96212288eed20e12fa6